scout/server/blueprints/cases/templates/cases/cases.html

```diff
@@ -222,6 +222,10 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
           {% set label_class = 'info' %}
         {% elif analysis_type == 'panel' %}
           {% set label_class = 'info' %}
+        {% elif analysis_type == 'mixed' %}
+          {% set label_class = 'danger' %}
```

Comment:
```suggestion
          {% set label_class = 'mixed' %}
```

Reply:
And add this code in styles.css:

/* Create a purple label to be used for spans, buttons etc. */
.label-mixed {
  background-color:#800080;
}

Doesn't have to be purple, but maybe it's nice if it's different from the others

Reply:
Nice suggestion, I fixed this and it works fine:)